### ISSUE TITLE
On windows tries to fetch a local package from npm

### ISSUE THREAD

**Issue by AlastairTaft** [lerna/lerna#503]
Works okay on Mac. If I run the same thing on windows it bombs out trying to fetch a package that exists in the local repository. It should probably just symlink it.

Here is the console log.

```cmd
C:\projects\reformed>npm install

> undefined install C:\projects\reformed
> lerna bootstrap

Lerna v2.0.0-beta.32
Bootstrapping 7 packages
Preinstalling packages
@luna/csv-to-json                                  ╣███████░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░╠
simple-react-starter-kit                           ╣█████████████░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░╠
@luna/reformed-util                                ╣████████████████████░░░░░░░░░░░░░░░░░░░░░░░░░░░╠
@luna/rwitl-components                             ╣███████████████████████████░░░░░░░░░░░░░░░░░░░░╠
@luna/screen-renderer                              ╣██████████████████████████████████░░░░░░░░░░░░░╠
@luna/web-demo                                     ╣████████████████████████████████████████░░░░░░░╠
demo-products                                      ╣███████████████████████████████████████████████╠
Installing external dependencies
NpmUtilities.installInDir     ("C:\\projects\\reformed\\packages\\editor", ["chalk@^1.1.3","gaze@^1.
0.0","source-map-support@^0.4.0...)
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Alastair\\AppData\\Roaming\\npm\\nod
e_modules\\npm\\bin\\npm-cli.js" "install" "chalk@^1.1.3" "gaze@^1.0.0" "source-map-support@^0.4.0"
"webpack-dev-server@^1.14.1" "@luna/screen-renderer@^0.12.2" "@luna/reformed-util@^0.12.2" "babel-cl
i@^6.7.7" "babel-core@^6.7.7" "babel-loader@^6.2.4" "babel-plugin-add-module-exports@^0.1.4" "babel-
polyfill@^6.7.4" "babel-preset-es2015@^6.6.0" "babel-preset-react@^6.5.0" "babel-preset-stage-0@^6.5
.0" "compression-webpack-plugin@^0.3.2" "connect-gzip-static@^1.0.0" "express@^4.13.4" "file-loader@
^0.8.5" "jss@^5.5.2" "jss-preset-default@^0.3.1" "react-redux@^4.4.5" "react-router@^2.3.0" "react-t
ap-event-plugin@*" "redux@^3.5.2" "redux-async-props@^0.1.10" "rimraf@^2.5.2" "url-loader@^0.5.7" "w
ebpack@^1.13.0"
npm ERR! node v6.2.1
npm ERR! npm  v3.8.6
npm ERR! code E404

npm ERR! 404 Not found : @luna/screen-renderer
npm ERR! 404
npm ERR! 404  '@luna/screen-renderer' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\projects\reformed\packages\editor\npm-debug.log
```

**Comment by AlastairTaft** [comment]
I managed to avoid the issue by updating all my package versions to the same number. 

I had a couple of packages with version 0.12.0 whereas the latest was 0.12.2, this was due to a merge from a branch back into master where master had had a couple of publishes. This didn't cause me any issue on Mac but it did on Windows.

I then hit this issue #504 but reverting to lerna beta30 side stepped that for me.
